apps/modernization-ui/src/components/button/link/LinkButton.tsx

```diff
@@ -5,20 +5,22 @@ type Props = {
     href: string;
     label?: string;
     type?: 'outline' | 'solid';
+    unstyled?: boolean;
```

Comment:
Suggestion: Might be cleaner to just pass a `className` and change the styling that way.

Reply:
There is that `type = 'outline` setting that's hard-coded in. I figured it cleanest to override with a prop but open to suggestions?